### PR TITLE
Fix need for meta-python and BSP recipe upgrades

### PR DESCRIPTION
**Pull request by kraj**

### CODE REVIEW

File: recipes-core/packagegroups/packagegroup-rpi-test.bb [hunk 1]
```diff
@@ -17,13 +17,16 @@ RDEPENDS_${PN} = "\
...
-    python3-sense-hat \
     connman \
     connman-client \
     wireless-regdb-static \
     bluez5 \
 "
 
+RDEPENDS_${PN} += "\
+    ${@bb.utils.contains("BBFILE_COLLECTIONS", "meta-python", "python3-sense-hat", "", d)} \
+"
```
Comment (pbrkr) [inline recipes-core/packagegroups/packagegroup-rpi-test.bb:28]: Could this be done via a `packagegroup-rpi-test.bbappend` file under `dynamic-layers/meta-python/...` to keep things grouped together?

Comment (agherzan) [reply recipes-core/packagegroups/packagegroup-rpi-test.bb:28]: I would also like that. It feels like we just break that package when meta-python is not available.

Comment (kraj) [reply recipes-core/packagegroups/packagegroup-rpi-test.bb:28]: Yes, I thought of that but this seems more directional when someone wanted to use this package that they would need meta-python, shoving all into dynamic-layers would make it more cryptic.

Comment (kraj) [reply recipes-core/packagegroups/packagegroup-rpi-test.bb]: moved now

**Review by agherzan — APPROVED** [review]
Looks good!

**Comment by agherzan** [comment]
@pbrkr Thumbs up?

**Comment by pbrkr** [comment]
@agherzan Thumbs up from me. I'm currently testing a few changes which I've based on top of this branch.

**Review by pbrkr — APPROVED** [review]

**Comment by agherzan** [comment]
Thanks, guys. Merged.

**Pull request merged**